Controller/Admin/Transaction/TransactionTabPostProcessing.php

```diff
@@ -13,14 +13,16 @@
 use OxidEsales\Eshop\Core\Exception\StandardException;
 
 use Wirecard\Oxid\Core\Helper;
+use Wirecard\Oxid\Core\PaymentMethodHelper;
 use Wirecard\Oxid\Core\TransactionHandler;
 use Wirecard\Oxid\Core\PaymentMethodFactory;
+use Wirecard\Oxid\Extend\Model\Payment;
```

Comment:
I don't see any use of this class

Reply:
You are right, I removed it and we will adapt our code sniffer